Created Introduction

natac13 opened the pull request:
A file to introduce myself to the Dojo, and declare my self to them

natac13 committed:
Created Introduction

A file to introduce myself to the Dojo, and declare my self to them

codeschool-kiddo commented on introduction.md:
```diff
@@ -0,0 +1,3 @@
+Hello Dojo Masters
```
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.

natac13 replied:
You bet!

natac13 committed:
Add favorite path

Was requested to add favorite path from Code School in the introduction.md file. I have made these changes.
